inventory/packages/yum.go

```diff
@@ -132,16 +99,19 @@ func YumUpdates() ([]PkgInfo, error) {
 	}
 
 	var pkgs []PkgInfo
-	for _, ln := range lines[1:] {
+	for _, ln := range lines {
 		pkg := strings.Fields(ln)
 		if len(pkg) == 2 && pkg[0] == "Obsoleting" && pkg[1] == "Packages" {
 			break
 		}
 		if len(pkg) != 3 {
-			DebugLogger.Printf("%s does not represent a yum update\n", ln)
 			continue
 		}
 		name := strings.Split(pkg[0], ".")
+		if len(name) != 2 {
+			DebugLogger.Printf("%s does not represent a yum update.", ln)
```

Comment:
nit: consider wrapping `%s` with single quotes to make it easier to parse the log. (below too)